### PR TITLE
Add styleguide preview page

### PR DESCRIPTION
**Pull request by lukaszklis**
Since we don't have a license bought for the font yet, we have to install the trial one locally and preview the page. Unfortunately now it seems to be working only in Safari (on my Mac) since the custom font is not saved to the system fonts and browsers refuse to load them due to the security policies they have. Just for a preview for Nicolas, this should be enough, will update it once we get a proper license with the web fonts attached.

### CODE REVIEW

**Comment by sfrdmn** [comment]
LGTM. BTW, I moved `/css` to `/assets/css` in my last pull in anticipation of having a Jekyll independent build step sometime later

**Comment by lukaszklis** [comment]
@sfrdmn thanks! Will update the PR. :)

File: _layouts/styleguide.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
```
Comment (Kriesse) [inline _layouts/styleguide.html]: Do we need this still? I think we can get rid of it.

File: styleguide.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+permalink: /styleguide
+font_types: ['thin', 'thin-oblique', 'regular', 'regular-oblique', 'medium', 'bold']
+---
+
+{% for type in page.font_types %}
+  <h1 class="styleguide__header styleguide__header--{{ type }}">One Morning, When Gregor Samsa Woke</h1>
+
+  <div class="styleguide__content styleguide__content--{{ type }}">
+    <p>
+      From troubled dreams, he found himself transformed in his bed into a horrible vermin. He lay on his armour-like back,
```
Comment (Kriesse) [inline styleguide.html:12]: 👍  on lorem ipsum choice

**Comment by Kriesse** [comment]
@lukaszklis Thanks, LGTM! Could you also put the font (and while it's working only locally, a reasonable fallback) on the current live page on 2017.cssconf.eu and sponsoring.cssconf.eu? `I am tired of font-family: fantasy`. Just any sans-serif fallback would be better :)

**Pull request merged**